src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -7,358 +7,694 @@ description: >
 
 <figure class="w-figure">
-  <img src="robot-specimen.png" alt="A specimen of Roboto">
-  <figcaption class="w-figcaption">
-    Examples of the Roboto typeface that was designed and developed
-    Christian Robertson.
+  <img src="roboto-family-specimen.png" style="max-width: 100%;" alt="A specimen composition and list of different styles of the Roboto family">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Left: a specimen of the Roboto typeface family. Right: named styles within the family.
   </figcaption>
 </figure>
 
+## Challenges for the Designer and Developer
+
+When a designer creates a print project they face some constraints, such as the
+physical size of the page layout, the number of colors they can use (which is
+determined by the kind of printing press that will be used), and so on. But they
+can use as many typeface styles as they like. This means that the typography of
+print media is often rich and sophisticated, so that the reading experience is
+truly delightful. Think of the last time you enjoyed browsing an excellent
+magazine.
+
+Web designers and developers have different constraints than print designers,
+and an important one is the associated bandwidth costs of our designs. This
+has been a sticking point for richer typographic experiences, as they come at
+a cost. With traditional web fonts, each style used in our designs requires
+users to download a separate font file, which increases latency and page
+rendering time. Only including the Regular and Bold styles, plus their italic
+counterparts, can amount to 500 KB or more of font data. This is even before
+we have dealt with how the fonts are rendered, the fallback patterns we need
+to use, or undesirable side-effects such as [FOIT and
+FOUT.](https://www.zachleat.com/web/fout-vs-foit/)
+
+Many font families offer a much wider range of styles, from Thin to Black
+weights, narrow and wide widths, a variety of stylistic details, and even
+size-specific designs (optimized for large or small text sizes.) Since you'd
+have to load a new font file for every style (or combinations of styles), many
+web developers choose not to use these capabilities, reducing the reading
+experience of their users.
+
+## Anatomy of a Variable Font
+
+Variable fonts address these challenges, by packing styles into a single file.
+
+This works by starting with a central or 'default' master style, usually the
+'Regular'–an upright roman design with the most typical weight and width that is
+most suitable for plain text. This is then connected to other style masters in a
+continuous range, called an 'axis.' The most common axis is **Weight**, which
+can connect the default style through to a Bold master style. Any individual
+style can be located along an axis, and is called an 'instance' of the variable
+font. Some instances are named by the font developer, for example Weight axis
+location 600 is called SemiBold.
```

Comment:
@RoelN is there any other phrase we can use other than `master` here? My org is asking me to avoid that word because of it's political/historical connotations (master/slave)

Reply:
Done!